packages/stream-metadata/src/node.ts

```diff
@@ -116,10 +117,11 @@ export function setupRoutes(srv: Server) {
 
 	// not cached
 	srv.get('/health', checkHealth)
+	srv.get('/refreshStatus/:invalidationId', fetchRefreshStatus)
 
 	// should be rate-limited, but not yet
 	srv.get('/space/:spaceAddress/refresh', { onResponse: spaceRefreshOnResponse }, spaceRefresh)
-	srv.get('/user/:userId/refresh', { onResponse: userRefreshOnResponse }, userRefresh)
```

Comment:
why did this change ^

Reply:
Oh, i see. Nice.

Reply:
We have the onResponse hook on the space refresh because we want to wait it to finish before invalidating opensea cache.
We don’t need to do this in the user refresh endpoint.

Reply:
Also, now that we want to return the invalidationId after calling user refresh, I had to move the invalidation from the hook to the endpoint itself. The hook was empty after it